migrations/tables/app.yaml

```
@@ -85,3 +85,5 @@ spec:
         default: 0
         constraints:
           notNull: true
+      - name: channel_id
```

how about renaming this to
```suggestion
      - name: selected_channel_id
```
this way it's clearer what this column represents. preferably rename it in the app type/struct as well.

---

Renamed in: https://github.com/replicatedhq/kots/pull/4767/commits/c39c4553a3b154a1bdfe09fdfea78df9464d3fd1